Add `unitPriceDisplay` and `displayListPrice` props to unit-price block (#51)
vtex-apps/product-list: docs/README.md

Therefore, in order to customize the Product List configuration, you can simply use the default implementation in your code and change it as you wish.

| Prop name              | Type   | Description                                                                                                                                   | Default value |
| ---------------------- | ------ | --------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |

> | `unitPriceDisplay`     | `Enum` | Determines if the unit price should always appear or appears only when the product quantity is bigger than one. (values: `always`, `default`) | `'default'`   |

Reviewer: ```suggestion
| `unitPriceDisplay`     | `Enum` | Defines whether the unit price should always be displayed (`always`) or only when the product quantity is bigger than one (`default`) | `default`   |
```

> | `displayUnitListPrice` | `Enum` | Defines whether the unit list price should be displayed when different from price (`always`) or not (`never`)    | `never`     |

Reviewer 2: Is there a reason to display the list price if the price is the same?

Author: @klzns No, list price will only be shown when it is different from price

Reviewer 2: So I guess we can remove this prop. What do you think?

Author: @klzns this property will show the list price when it is different from the price, it is currently not shown. Maybe the documentation is causing some confusion

Reviewer 3: @klzns I think this prop is important because of the current stores that not implement this HTML element. Think: If the store customers don't have this element and we don't allow them to  able and disable this element, this will broke some layouts.
Maybe its better to create another block? o,o

> | `displayUnitListPrice` | `Enum` | Display the unit list price when it's different from price. (values: `'always'` or `'never'`)                                                 | `'never'`     |

Reviewer: ```suggestion
| `displayUnitListPrice` | `Enum` | Defines whether the unit list price should be displayed when different from price (`always`) or not (`never`)    | `never`     |
```

> | `textAlign`            | `Enum` | Enable to set a text alignment. The possible values are: `left`, `center` or `right` )                                                                  | `undefined`          |

Reviewer 4: I don't like much these empty spaces in the table, as it is really cumbersome to edit them manually, and with long lines as these they don't even match up in the editor (only if you have a _really_ wide screen). Can you remove them?

```suggestion
| Prop name | Type | Description | Default value |
| --- | --- | --- | --- |
| `unitPriceDisplay` | `Enum` | Defines whether the unit price should always be displayed (`always`) or only when the product quantity is bigger than one (`default`) | `default` |
| `displayUnitListPrice` | `Enum` | Defines whether the unit list price should be displayed when different from price (`always`) or not (`never`) | `never` |
| `textAlign` | `Enum` | Enable to set a text alignment. The possible values are: `left`, `center` or `right` ) | `undefined` |
```

> | `textAlign`            | `Enum` | Enable to set a text alignment. (values: `'left'`. `'center'` or `'right'` )                                                                  | `''`          |

Reviewer: ```suggestion
| `textAlign`            | `Enum` | Enable to set a text alignment. The possible values are: `left`, `center` or `right` )                                                                  | `undefined`          |
```

### `unit-price` props

| Prop name   | Type     | Description                               | Default value |
| ----------- | -------- | ----------------------------------------- | ------------- |
| `textAlign` | `string` | Product unit prices position on the list. | `left`        |

> | `unitPriceDisplay` | `string` | Defines whether the unit price will always appear (`always`) or only when the quantity of products is greater than one (`default`). | `default`        |
> | `displayUnitListPrice` | `string` | Defines whether the list-price will appear when it is different from the price (`showWhenDifferent`) or if it should never appear (`notShow`). | `left`        |

Reviewer: ```suggestion
| `unitPriceDisplay` | `enum` | Defines when the unit price should be displayed. Possible values are: `always` (unit price is always displayed) or `default` (unit price is only displayed when the number of products is greater than one). | `default`        |
| `displayUnitListPrice` | `enum` | Defines whether the product list price should be displayed or not. Possible values are: `showWhenDifferent` (list price is displayed when it is different from the regular price) or`notShow` (list price is never displayed). | `notShow`        |
```

Reviewer: I'm not sure it `notShow` is the default value for `displayUnitListPrice` prop.

Author: @guerreirobeatriz suggestion applied. Actually the unit list price does not shows up, this pull request implements it.
